src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/ServiceHubLifetimeManagerFactory.cs

```diff
@@ -54,5 +56,14 @@ public ServiceHubLifetimeManagerFactory(IServiceProvider sp, IOptions<ServiceMan
                 default: throw new InvalidEnumArgumentException(nameof(ServiceManagerOptions.ServiceTransportType), (int)_options.ServiceTransportType, typeof(ServiceTransportType));
             }
         }
+
+        private void CheckHubProtocols()
+        {
+            var protocols = _serviceProvider.GetServices<IHubProtocol>().ToArray();
+            if (protocols.Length > 1 || protocols.Where(p => p.Name.Equals(Constants.Protocol.MessagePack)).Any())
```

Comment:
```suggestion
            if (protocols.Length > 1 || protocols.Any(p => p.Name.Equals(Constants.Protocol.MessagePack)))
```
case sensitive or insensitive here?